observation/interferometric_pointing.py

```diff
@@ -22,10 +22,10 @@ class NoTargetsUpError(Exception):
 parser.add_option('-m', '--max-duration', type='float', default=None,
                   help='Maximum duration of the script in seconds, after which script will end '
                        'as soon as the current track finishes (no limit by default)')
-parser.add_option('--max-extent', type='float', default=1.0,
-                  help='Maximum extent in degrees, the script will scan ')
-parser.add_option('--number-of-steps', type='int', default=10,
-                  help='Number of pointings to do while scaning , the script will scan ')
+parser.add_option('--max-offset', type='float', default=1.0,
```

Comment:
Oops, forgot about this one...

Since the default interferometric pointing SB does not set `--max-extent` / `--max-offset`, it's probably better to pick a sensible default value like 0.5 degrees. (Ideally this should be linked to the centre frequency.)

Reply:
i agree but insist that we keep such a change for another pull request!

for reference, the meerkat antenna half power point is ~1.27*lambda/13.965 [radian] ~5.2*lambda [deg]. i have recently suggested to sean & tony that we should use the following defaults:
- UHF-band: 2.5deg
- L-band: 1.5deg

Reply:
Sure.

**(Off-topic but couldn't resist!):**

So are you saying the equivalent dish diameter is 13.965 m?

This factor seems very close (within 0.7%) to the factor I use by default:
1.22 lambda / 13.5 [radian] = 5.18 lambda [deg]

The random beam fits I did seemed to need a lower value, but I've not quantified it yet, so I'm mildly surprised that the default is OK.